protocols/virtualfund/virtualfund.go

```diff
@@ -667,8 +667,17 @@ func (o *Objective) proposeLedgerUpdate(connection Connection, sk *[]byte) (prot
 		return protocols.SideEffects{}, fmt.Errorf("error finding a supported state: %w", err)
 	}
 
+	proposed, proposedFound := ledger.Proposed()
+
```

Comment:
with respect to ledger syncing and to possible future batch updates to ledgers, this approach may not be correct.

EG: a hub may process N virtualfund objectives at once, update their own local copy of the ledger channel with subsequent proposals that build on top of the existing unanswered proposal, and send only the "latest" on over the wire.

That said, I think this is fine here for now.